### DISCUSSION TITLE
SSR crashed with Sentry

### DISCUSSION THREAD

**Discussion by jinho-youha** [vercel/vercel#8489] [Help]
Hi, suddenly my app got the crash (500 errors) with Sentry configuration. 
I cannot reproduce this on my local machine. What shall I need to check? 

I just found 500 errors in Chrome development tool. But no log from vercel output. 

Next.js: 12.2.3
Sentry SDK(@sentry/nextjs): 7.11.1

**Comment by amyegan** [comment]
Hi @jinho-youha! If you are getting an error from a Serverless Function, then you should be able to [view the function logs from your Dashboard](https://vercel.com/guides/where-are-vercel-logs). That should reveal more about what is causing the error.

I can help you debug more if you are able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or your repository. More information from the 500 error would also be helpful for debugging.
